ui/src/api/fetch-func/user.ts

```diff
@@ -0,0 +1,7 @@
+import ApiService from '../index';
+import Paths from '../paths';
+
+export async function getUser() {
+  const res = await ApiService.getAuthApi().fetchUser(Paths.USER);
```

Comment:
[2] I don't like that we pass the url as a param here. I think it should be specified on a service level
Might wanna operate with userId here